### PR TITLE
Fix: Fixed the broken link to contributing.md and also linked the GitHub actions badge to main branch

### PR DESCRIPTION
**Pull request by hasnainmakada-99**
### Checklist
- [x] Code compiles correctly
- [x] Created tests which fail without the change (if possible)
- [x] All tests passing
- [x] Extended the README/documentation, if necessary

#### What does this PR do?
- Made changes in the readme.md file

#### Where should the reviewer start?
- From the Readme.md file
#### What is missing?
- The link to contributing.md file
#### How should this be manually tested?
- none
#### Any background context you want to provide?
- none
#### What are the associated tickets?
- No
[PG-XX](https://napptive.atlassian.net/browse/PG-XX)

### CODE REVIEW

**Comment by hasnainmakada-99** [comment]
@Carmendelope @dhiguero please review this PR

File: README.md [hunk 1]
```diff
@@ -47,11 +47,12 @@ RUN_INTEGRATION_TEST=all make test
 
 This repository is integrated with GitHub Actions.
 
-![Check changes in the Main branch](https://github.com/napptive/catalog-manager/workflows/Check%20changes%20in%20the%20Main%20branch/badge.svg)
+<a href="https://github.com/hasnainmakada-99/catalog-manager.git">![Check changes in the Main branch](https://github.com/napptive/catalog-manager/workflows/Check%20changes%20in%20the%20Main%20branch/badge.svg)
```
Comment (dhiguero) [inline README.md:50]: Please remove this change as the badge is not intended to be a link, just to show the status. Moreover, the links should point to the main repo, not to a fork.

**Review by dhiguero — CHANGES_REQUESTED** [review]
Thanks for the contribution, please review the comments.

**Comment by hasnainmakada-99** [comment]
Ok @dhiguero I'll make the changes as you requested

**Comment by hasnainmakada-99** [comment]
Hey @dhiguero I've made the changes as you requested, Please review it when you are free

**Review by dhiguero — APPROVED** [review]

**Pull request merged**